NitroxServer-Subnautica/Program.cs

```diff
@@ -83,6 +83,12 @@ private static void Main(string[] args)
         private static Assembly CurrentDomainOnAssemblyResolve(object sender, ResolveEventArgs args)
         {
             string dllFileName = args.Name.Split(',')[0];
+
+            if (dllFileName.Contains(".resources"))
+            {
+                return null;
+            }
```

Comment:
Can this happen ?

Reply:
And if not the users will find a way 🙃

Reply:
If it didn't happen- what prompted you to add it?

Reply:
That's from kill I believe

Reply:
embedded resources inside dll

Reply:
embedded resources in dll